index.html

```diff
@@ -166,5 +216,33 @@ <h3>LET'S CONNECT</h3>
                 <br>
             </ul>
 </section>
-</body>
-</html>
\ No newline at end of file
+<footer>
+    <form action="https://formspree.io/f/xrgdkyvk" method="post">
+        <header>
+          <legend>Contact me</legend>
+          <p>
+            If you have an application you <br>
+            are interested in developing, a <br>
+            feature that you need built or a <br>
+            project that needs coding. I’d <br>
+            to help with it
+          </p>
+        </header>
+        <ul style="list-style: none">
+          <li>
+            <input type="text" name="user-name" id="name" minlength="5" placeholder="Yeremias" required maxlength="30">
+          </li>
+          <li>
+            <input type="email" name="email" id="email" placeholder="natajayanj14@gmail.com" required>
+          </li>
+          <li>
+            <textarea name="user_message" id="msg" placeholder="Write your message here" maxlength="300" required></textarea>
```

Comment:
- [ ] As written in the project requirements. I kindly suggest that you increase your text area maxlength to 500 because the Message field should accepts up to 500 chars